pipeline/ingestor/ingestor_common/src/test/scala/uk/ac/wellcome/platform/ingestor/common/fixtures/IngestorFixtures.scala

```diff
@@ -34,7 +34,7 @@ trait IngestorFixtures
         withBigMessageStream[T, R](queue) { messageStream =>
           val ingestorConfig = IngestorConfig(
             batchSize = 100,
-            flushInterval = 5 seconds
+            flushInterval = 1 seconds
```

Comment:
Why change this?

Reply:
Because these tests were really very sloooooooowwwwwwwwwwwwwwwww and this sped them up a bit (still quite slow though, I spent some time looking into speeding them up and concluded that it was kind of intrinsic to what they're doing)